storage/core/src/server/internal/Helpers.ts

```diff
@@ -121,12 +125,14 @@ export async function streamToTransferType(
 export async function downloadFromUrl(
   input: UrlDownloadInput
 ): Promise<TransferData> {
-  const { transferType, url } = input;
+  const { transferType, url, abortSignal } = input;
 
   // There is an issue with Axios type definitions. Casting should be removed
   // after upgrading to Axios 1.0
   // See: https://github.com/axios/axios/pull/4229
-  const signal = input.abortSignal as AbortSignal | undefined;
+  const signal = abortSignal
+    ? (createClientAbortSignal(abortSignal) as unknown as AbortSignal)
```

Comment:
Is `as unknown` casting needed?

Reply:
Removed it. It's unnecessary right now, but would be required if I modified the AbortSignal wrapper.